apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -0,0 +1,167 @@
+                    <th>QID</th>
+                    <th>Auto Points</th>
+                    <th>Manual Points</th>
+                    <th colspan="2">Submissions to grade</th>
+                    <th>Grading assigned to</th>
+                    <th>Graded by</th>
+                    <th>Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  ${questions.map(
+                    (question) =>
+                      html` <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                              .id}/manual_grading/assessment_question/${question.id}"
+                          >
+                            ${question.alternative_group_number}.${question.alternative_group_size ===
+                            1
+                              ? ''
+                              : `${question.number_in_alternative_group}.`}
+                            ${question.title}
+                          </a>
+                        </td>
+                        <td>${question.qid}</td>
+                        <td class="text-center">
+                          ${question.max_auto_points
+                            ? resLocals.assessment.type === 'Exam'
+                              ? (question.points_list || [question.max_manual_points ?? 0])
+                                  .map((p) => p - (question.max_manual_points ?? 0))
+                                  .join(',')
+                              : (question.init_points ?? 0) - (question.max_manual_points ?? 0)
+                            : '—'}
+                        </td>
+                        <td class="text-center">${question.max_manual_points || '—'}</td>
+                        <td class="text-center" data-testid="iq-to-grade-count">
+                          ${question.num_instance_questions_to_grade} /
+                          ${question.num_instance_questions}
+                        </td>
+                        <td>
+                          ${progressBar(
+                            question.num_instance_questions_to_grade,
+                            question.num_instance_questions,
+                          )}
+                        </td>
+                        <td>
+                          ${question.num_instance_questions_assigned > 0
+                            ? html`<strong class="bg-warning rounded px-1">
+                                  ${resLocals.authz_data.user.name ??
+                                  resLocals.authz_data.user.uid}</strong
+                                >${question.assigned_graders?.some((u) =>
+                                  idsEqual(u.user_id, resLocals.authz_data.user.user_id),
+                                )
+                                  ? ', '
+                                  : ''}`
+                            : ''}
```

Comment:
I find this quite difficult to read in its current form. This is slightly better:

```suggestion
                            ? html`
                                <strong class="bg-warning rounded px-1">
                                  ${resLocals.authz_data.user.name ?? resLocals.authz_data.user.uid}
                                </strong>
                                ${question.assigned_graders?.some((u) =>
                                  idsEqual(u.user_id, resLocals.authz_data.user.user_id),
                                )
                                  ? ', '
                                  : ''}
                              `
                            : ''}
```

It's also not clear why we add a trailing command if `assigned_graders` contains the current user. Wouldn't we want to add that comma if there are *other* users that are assigned?

Reply:
On the phone now, will give a proper answer later, but one issue with moving the tags around like you suggested is that it adds a space inside the strong tag, before the comma.

Reply:
> It's also not clear why we add a trailing command if `assigned_graders` contains the current user. Wouldn't we want to add that comma if there are _other_ users that are assigned?

You were right. Since this is done again in the next line I moved it to a variable and used that instead.

Reply:
> I find this quite difficult to read in its current form. This is slightly better:

Done in 62d97cd7f, though I kept the closing tag as is to ensure there is no space between the end of the username and the comma that succeeds it.